setup.py

```diff
@@ -5,4 +5,19 @@
 
 import setuptools
 
-setuptools.setup()
+try:
+    import tomllib
+except ImportError:
+    import tomli as tomllib  # type: ignore[no-redef]
+with open("pyproject.toml", "rb") as f:
+    _metadata = tomllib.load(f)["project"]
+
+
+setuptools.setup(
```

Comment:
We should add a comment here indicating that this code exists to work around a Github limitation (and link to the discussion at https://github.com/community/community/discussions/6456).

Reply:
Good point. I was looking at the exact same one and want to try the workaround from the Flask team.